lib/commands/screenshots.js

```diff
@@ -45,26 +52,42 @@ commands.getScreenshot = async function () {
     }
     return data;
   };
+
   try {
-    return await getScreenshotFromWDA();
+    if (this.isRealDevice() && await isIdevicescreenshotAvailable()) {
+      log.debug(`Taking screenshot with 'idevicescreenshot'`);
+      return await getScreenshotWithIdevicelib(this.opts.udid);
+    } else {
```

Comment:
`else` is not needed here

Reply:
That's true, but it _is_ an else condition (i.e., it is the alternative to the original). I prefer to keep it in this case.